network/activate.go

```diff
@@ -28,6 +28,9 @@ func (n *network) payloadFromConnections(behaviorID string, queue []spec.Network
 	// storage. Here it is important to obtain the order of the behavior IDs
 	// stored as connections. They represent the input interface of the requested
 	// CLG.
+	//
+	// TODO there needs to be some sort of variation when executing existing CLG trees
```

Comment:
We need to add variations to the selection process of network payloads being merged together. The requested CLG needs to be executed with different inputs from time to time, regardless known successful configurations.